helene/reqs.md

```diff
@@ -1,49 +1,129 @@
-#Global Requirements
+# Global Requirements
 This document's aim is to identify needs and requirements from all the Openstax products that are using a common html file and common CSS styling
 
-###General needs
-Take specific formats into account 
-  - webview
-  - epub
-  - pdf 
-  - Tutor
-  
-Global identity, consistent look and feel
-  - format specific but not disconnected from other products
-  
-Cross team friendly
-  - accessible and efficiently accessed/modified by all teams
-
-##Product Specific
-###PDF styles/framework
-Maintainablity: 
-  - lifetime
-    - retain control over natural evolution of code base over time
-    - scalable
-  - multiple users, code consistency
-  - future development (features, refactor, new designs)
-  - change in data structure (new DOM)
-  
-Ease of use: 
-  - efficiency: clear file structure, clear logic pathways: limit time wasted figuring out what does what and what is where
-  - access to PDF(print) specific needs: pagination, folio
-  - minimal regression testing: ablitity to add or modifiy content without having to do a full stack regression testing each time
-  
-Fast creation
-  - DRY code
-  - reusable/modular when creating/updating templates
-  - ability to add various or new features easily
-  - turn around issues or bugs quickly
- 
--------- 
-##General philosophy/concepts 
-
-###For cleaner/better code
+### General needs
+
+As a **student/instructor** I want the different formats (webview, epub, pdf, tutor) to be similar enough to do my homework and reading assignments easily.
+
+As **Openstax** I want the styling to be similar so there is brand identity and a consistent "look and feel" within the constraints of the various digital formats.
+
+As a **developer** I want the code to be easily approachable and efficiently accessed/modified by all teams so I can make changes faster.  
+
+# Product Specific
+
+## PDF styles/framework
+
+### Maintenance
+
+As a **developer** I want the code to scale to multiple books so adding a new book is simple.
+
+As a **developer** I want to retain control over the evolution of the codebase over time _(PHILS: I am not sure what is meant by this)_
```

Comment:
I rephrased this as "I want to easily refactor the code so it can evolve over time"